### ISSUE TITLE
Element explorer hangs in protractor >=1.7 if you have node >=0.10.31

### ISSUE THREAD

**Issue by hankduan** [angular/protractor#1890]
Element explorer hangs in versions 1.7 or later. 

NOTE: this behavior only appears if you are on node version 0.10.31 or above

Under closer look, you should see that this delay is not indeterministic. To be exactly:
if you wait n seconds to enter a command, the delay will be (n + time to execute command) seconds. 
In other words, if you execute a simple command like `var x = 3`, which takes no time to execute, the delay is equal to the amount of time you waited before entering the command. 

Try this:
```
$ //start element explorer
> var x = 3  //should return immediately
> // now wait 10 seconds, and then type `var x = 3`
> // should wait ~10s before returning. 
```

An issue has been filed with additional information here: https://github.com/joyent/node/issues/9333

**Comment by hankduan** [comment]
FYI: The only workaround for this issue right now is either:
1) downgrade protractor to 1.6.1
or
2) use node.js to 0.10.30 or lower.

**Comment by shattenjager** [comment]
I tried to downgrade to 0.10.29 but still is not working. I get SyntaxError: evalmachine.<anonymous>:1 on my Win machine. I also tried to install an older version of Protractor but I don't know how, since I use npm. My current version is 2.1.0. Also I get this error not using elementExplorer but the debug mode triggered by a browser.pause() method in my code, then I used "repl" to switch to the interactive mode; from there I can do only a browser.get() but can't manage to interact with any element.
![image](https://cloud.githubusercontent.com/assets/7750660/7850639/e5f84dc2-0499-11e5-8d15-2d03f75bd79e.png)

Win  8.1 64
node 0.12.4
npm 2.10.1
protractor 2.1.0

**Comment by juliemr** [comment]
@shattenjager you do have a syntax error, you're missing a `)` closing the `element` call.

I'm not sure why your error message is so bad, that seems like something to do with how node interacts with windows. On mac, it prompts you to continue the input, since the parentheses haven't been matched.

**Comment by shattenjager** [comment]
Geez, thanks, I feel dumb, I apologize!!

**Comment by hankduan** [comment]
fyi: This will be fixed when node 0.12.5 gets released. https://github.com/joyent/node/issues/9333#event-333488911.

**Comment by juliemr** [comment]
Node 0.12.5 is out, woo! Closing.
